sql/moz-fx-data-shared-prod/telemetry_derived/sponsored_tiles_clients_daily_v1/query.sql

```diff
@@ -62,6 +63,66 @@ desktop_activity_stream_events AS (
     `moz-fx-data-shared-prod.activity_stream.events`
   WHERE
     DATE(submission_timestamp) = @submission_date
+    AND CAST(metadata.user_agent.version AS INT64) < 123
+  GROUP BY
+    1,
+    2
+),
+--- Current telemetry for dismissals and deactivations comes in Glean's newtab ping as of Fx120 (Nov 21, 2023)
+desktop_newtab_events AS (
+  SELECT
+    n.client_info.client_id AS client_id,
+    n.metrics.uuid.legacy_telemetry_client_id AS legacy_telemetry_client_id,
+    DATE(submission_timestamp) AS submission_date,
+    COUNTIF(
+      e.category = 'topsites'
+      AND e.name = 'dismiss'
+      AND `mozfun.map.get_key`(e.extra, 'is_sponsored') = 'true'
+    ) AS sponsored_tiles_dismissal_count,
+    COUNTIF(
+      e.name = 'pref_changed'
+      AND `mozfun.map.get_key`(
+        e.extra,
+        'pref_name'
+      ) = 'browser.newtabpage.activity-stream.showSponsoredTopSites'
+      AND `mozfun.map.get_key`(e.extra, 'new_value') = 'false'
+    ) AS sponsored_tiles_disable_count
+  FROM
+    `moz-fx-data-shared-prod.firefox_desktop.newtab` n,
+    UNNEST(events) e
+  WHERE
+    DATE(submission_timestamp) = @submission_date
+    AND `mozfun`.norm.browser_version_info(client_info.app_display_version).major_version >= 123
+  GROUP BY
+    1,
+    2,
+    3
+),
+desktop_joint_events AS (
+  SELECT
+    n.submission_date,
+    n.legacy_telemetry_client_id AS client_id,
+    n.sponsored_tiles_dismissal_count,
+    n.sponsored_tiles_disable_count
+  FROM
+    desktop_newtab_events n
+  UNION DISTINCT
```

Comment:
It shouldn't be possible to have any overlap between these two since we're filtering by version, so I would make this a UNION ALL

```suggestion
  UNION ALL
```